src/Controls/tests/DeviceTests/Elements/Border/BorderTests.cs

```diff
@@ -84,7 +84,8 @@ public async Task BorderStrokeThicknessDoesNotInsetStrokePath()
 #endif
 		}
 
-
+		// NOTE: this test is slightly different than MemoryTests.HandlerDoesNotLeak
+		// It adds a child to the Border, which is a worthwhile test case.
 		[Fact("Border Does Not Leak")]
 		public async Task DoesNotLeak()
```

Comment:
Should we make a special general for everything that takes a Child ? and it would catch more leaks? Like ScrollVIew with Content, Refreshivew with a Empttyview etc.. ?!